toolchain/check/testdata/facet/no_prelude/fail_todo_call_combined_impl_witness.carbon

```diff
@@ -66,28 +66,14 @@ fn G[T:! A & Empty & B](t: T) {
   // CHECK:STDERR:   ^~~~~~~~
   // CHECK:STDERR:
   T.(A.AA)();
-  // CHECK:STDERR: fail_todo_call_combined_impl_witness.carbon:[[@LINE+7]]:3: error: cannot access member of interface `B` in type `Empty & A & B` that does not implement that interface [MissingImplInMemberAccess]
+  // CHECK:STDERR: fail_todo_call_combined_impl_witness.carbon:[[@LINE+4]]:3: error: cannot access member of interface `B` in type `Empty & A & B` that does not implement that interface [MissingImplInMemberAccess]
```

Comment:
Why do we still get these errors?

Reply:
After rebasing on your recent work, the errors are different:

```
  // CHECK:STDERR: fail_todo_call_combined_impl_witness.carbon:[[@LINE+7]]:3: error: cannot implicitly convert value of type `Empty & A & B` to `A` [ImplicitAsConversionFailure]
  // CHECK:STDERR:   T.(A.AA)();
  // CHECK:STDERR:   ^~~~~~~~
```

Not sure why - I can look into it with another PR?

Reply:
Another PR is fine.

Reply:
These now pass in https://github.com/carbon-language/carbon-lang/pull/5085